src/Extension/FluentExtension.php

```diff
@@ -1512,4 +1601,38 @@ protected function localisedCopyNeeded(): bool
         // all other cases should not duplicate (normal edits)
         return false;
     }
+
+    /**
+     * Get a list of locale codes that represent locales that the model is localised in
+     * TODO make this method public and use $owner instead as this is a useful utility method
+     *
+     * @param DataObject $model
+     * @return array
```

Comment:
```suggestion
     * Get a list of locale codes that represent locales that the model is localised in
```
TODO comments in code get forgotten. Raise an issue instead so it can be tracked, discussed, and actioned.

PHPDocs aren't needed if they only provide the same information the actual typehints provide. They're likely to get stale if the actual hints change.

Reply:
All good, raised https://github.com/tractorcow-farm/silverstripe-fluent/issues/942 to cover this.